data/data-pipeline/data_pipeline/etl/sources/doe_energy_burden/etl.py

```diff
@@ -61,11 +61,6 @@ def transform(self) -> None:
             }
         )
 
-        # Convert energy burden to a fraction, since we represent all other percentages as fractions.
```

Comment:
Referencing this source [data](https://app.zenhub.com/files/358242767/627c0897-e4a9-4058-9641-d35b0d1fc1ea/download) placed by @BethMattern  - BURDEN values for each tract area already a fractions